augur/io/file.py

```diff
@@ -32,8 +34,18 @@ def open_file(path_or_buffer, mode="r", **kwargs):
     kwargs['encoding'] = ENCODING
 
     if isinstance(path_or_buffer, (str, os.PathLike)):
-        with xopen(path_or_buffer, mode, **kwargs) as handle:
-            yield handle
+        try:
+            with xopen(path_or_buffer, mode, **kwargs) as handle:
+                yield handle
+        except UnicodeDecodeError as e:
```

Comment:
Consider catching and handling `UnicodeEncodeError` too since this function is used for writing as well. (Right?)

Reply:
Yes, it's used for writing as well. When could this happen though? I'd think that any `UnicodeEncodeError`s are actual bugs and not user error.

Reply:
Ah, I guess it wouldn't, if we plan to never encode to anything other than a Unicode encoding. And since encoding in this function is force-set to `utf-8` rather than a default which can overridden by the caller, that won't happen unless the code changes.